merkledag/README.md

```diff
@@ -0,0 +1,144 @@
+# The merkledag
+
+
+Authors: [Juan Benet](github.com/jbenet)
+
+Reviewers:
+
+- [Jeromy Johnson](github.com/whyrusleeping)
+
+* * *
+
+This is the [Spec](../) for the IPFS Merkledag.
+
+## Definitions
```

Comment:
It's useful to have definitions for a spec in the same doc cause definitions can afford to be as specific as needed for this doc. a glossary one would be more general.

agree these (or at least some) should go into the glossary